languages/php.php

```diff
@@ -368,6 +459,48 @@ class ClassWithLogger
 }
 
 
+/**
+ * Enums (PHP >=8.1)
+ * https://www.php.net/manual/fr/language.types.enumerations.php
```

Comment:
English is better: https://www.php.net/manual/en/language.types.enumerations.php